frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineServerActions.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownSeparator,
+  DropdownToggle,
+  KebabToggle,
+  Tooltip,
+} from '@patternfly/react-core';
+import { DeleteServerModal, ViewServerModal } from '~/concepts/pipelines/context';
+
+type PipelineServerActionsProps = {
+  type?: string;
```

Comment:
because type can really only be two things, i would constrain the type to an enum or simply `"kebab" | "dropdown"`

If someone wanted to use this component elsewhere, they wouldn't know what to put in type without digging through the component

Reply:
Since you'll be updating this, I suggest using `variant` instead of `type` as the prop name to align with PF naming conventions.

Reply:
Done